pkg/cloud/scope/managedcontrolplane.go

```diff
@@ -267,3 +267,8 @@ func (s *ManagedControlPlaneScope) ImageLookupOrg() string {
 func (s *ManagedControlPlaneScope) ImageLookupBaseOS() string {
 	return s.ControlPlane.Spec.ImageLookupBaseOS
 }
+
+// SecretBackend returns the secret backend to use when looking up AMIs
```

Comment:
> // SecretBackend returns the secret backend to use when looking up AMIs

That doesn't seem right to me as a description? The AMI lookup is either EC2 API or SSM?